src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,84 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
+            <ul class="mint-dropdown__list">
+                <li>
+                    <a class="mint-dropdown__element" href="#">Option 1</a>
```

Comment:
they do not have to be link but main usage for these dropdowns are filters, and filters have to be links, i.e.  `/subject/maths` and so on.